gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala

```diff
@@ -5,98 +5,80 @@ import io.gatling.http.Predef._
 
 class BrowseAndListenSimulation extends Simulation {
 
-  val frontendUrl = "https://wayland-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
-  val backendUrl = "https://cms-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
+  val frontendUrl = sys.env.getOrElse("FRONTEND_URL", "https://wayland-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk")
+  val backendUrl = sys.env.getOrElse("BACKEND_URL", "https://cms-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk")
+  val numberOfConcurrentUsers = sys.env.getOrElse("CONCURRENT_USERS", 5).asInstanceOf[Int]
+  val rampUpOver = sys.env.getOrElse("RAMP_UP_OVER_SECONDS", 5).asInstanceOf[Int]
 
   val httpProtocol = http
     .baseUrl(frontendUrl)
     .doNotTrackHeader("1")
-    .acceptHeader("text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8") // Here are the common headers
+    .acceptHeader("text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8")
     .acceptLanguageHeader("en-US,en;q=0.5")
     .acceptEncodingHeader("gzip, deflate")
     .userAgentHeader("Mozilla/5.0 (Macintosh; Intel Mac OS X 10.8; rv:16.0) Gecko/20100101 Firefox/16.0")
     .inferHtmlResources()
 
-  val scn = scenario("Scenario Name")
-    .exec(
-      http("Visit the home page")
-        .get("/")
+  object Page {
+    def visit(description: String, url: String) = exec(
+      http(description)
+        .get(url)
         .check(status.is(200)) 
     )
-    .pause(4)
-    .exec(
+  }
+
+  object VideoPage {
+    def visit(url: String) = exec(
       http("Go to a video page")
-        .get("/content/3949")
-        .check(css("video source", "src").saveAs("videoLink"))
+        .get(url)
+        .check(css("video source", "src").saveAs("videoUrl"))
         .check(status.is(200))
     )
     .pause(2)
     .exec(
       http("Play the video")
-        .get("${videoLink}")
-        // .header("Range", "bytes=0-1023") 
+        .get("${videoUrl}")
     )
-    .pause(5)
-    .exec(
-      http("Return to the homepage")
-        .get("/")
+  }
+
+  object AudioPage {
+    def visit(url: String) = exec(
+      http("Go to a audio page")
+        .get(url)
         .check(status.is(200))
+        .check(css("audio source", "src").saveAs("audioUrl"))
     )
     .pause(2)
     .exec(
-      http("Browse all topics")
-        .get("/topics")
-        .check(status.is(200))
-    )
+      http("Play the audio")
+        .get("${audioUrl}")
+    )   
+  }
+
+  val scn = scenario("Scenario Name")
```

Comment:
I suppose these could be stuck in an array or something so we can have multiple, or pull them in from an include? I dunno just thinking.

Reply:
Moved to supports, we have reusable Page helpers